FirebaseMessaging.podspec

```diff
@@ -53,8 +53,8 @@ device, and it is completely free.
     unit_tests.source_files = 'Example/Messaging/Tests/*.[mh]'
     unit_tests.requires_app_host = true
     unit_tests.pod_target_xcconfig = {
-      # Unit tests do library imports using Firebase/Messaging relative paths.
-      'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}"/Firebase/Messaging',
+      # Unit tests do library imports using repo-root relative paths.
+      'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}"',
```

Comment:
Should be on the library build rule, not the testspec - like https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseABTesting.podspec#L40

Reply:
That was my first attempt, but didn't work either. I moved it to the library build now.

Reply:
Think I got it fixed now after I tuned the header search path in xcode a bit too.